src/features/content-answers/ContentAnswersModalFooterActions.scss

```diff
@@ -0,0 +1,19 @@
+@import '../../styles/variables';
+
+.bdl-ContentAnswersModalFooterActions {
+    position: relative;
+    top: -$bdl-grid-unit * 4;
+    right: 0;
+    left: 0;
+    align-self: flex-start;
+    width: 100%;
+    padding: $bdl-grid-unit * 3 0 0 0;
+}
+
+.bdl-ContentAnswersModalFooterActions-Button {
+    display: flex;
+    align-items: center;
+    min-height: $bdl-grid-unit * 10;
+    margin: auto;
+    padding: 0 $bdl-grid-unit * 4;
```

Comment:
i think if you pass `size="large"` to the button component then you wouldn't need these styles?

Reply:
looks like it default to large but the styles didnt seem to make a difference so removing them